src/main/java/campus/Campus.java

```diff
@@ -0,0 +1,41 @@
+package campus;
+
+import campus.infrastructure.Parser;
+import campus.infrastructure.Storage;
+import campus.infrastructure.TaskList;
+import campus.infrastructure.Ui;
+
+/**
+ * Contains the logic for the ChatBot named 'Campus'
+ */
+public class Campus {
+    private final Ui ui;
+    private final Parser parser;
+
+    /**
+     * Creation of a Campus instance
+     */
+    public Campus() {
+        String filePath = "src/main/java/campus/dataTest.txt";
```

Comment:
Should this be in SCREAMING_SNAKE_FONT?
```suggestion
        String FILE_PATH= "src/main/java/campus/dataTest.txt";
```